### PR TITLE
fix: change default value of `preview_max_dimensions` to 6016x6016

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by jnweiger — APPROVED** [review]
LGTM

**Comment by phil-davis** [comment]
Changelog?

I guess there should be something, because there has been a bit of discussion about this stuff. So it would be nice if the changelog at each release point shows clearly what the default setting is at each release.

**Review by phil-davis — APPROVED** [review]

**Comment by jnweiger** [comment]
Related #41193

File: lib/private/Preview/Image.php [hunk 1]
```diff
@@ -75,17 +75,17 @@ private function validateImageDimensions(\OC_Image $image): bool {
 	}
 
 	private function getMaxDimensions(): array {
-		// 24 MP - 6016 x 4000
-		$maxDimension = \OC::$server->getConfig()->getSystemValue('preview_max_dimensions', '6016x4000');
+		// 24 MP - 6016 x 6016
```
Comment (pako81) [inline lib/private/Preview/Image.php:78]: Should it not be 36 MP? But since it is just a comment, maybe not that relevant..

Comment (DeepDiver1975) [reply lib/private/Preview/Image.php:78]: 24 mega pixel is 6k x 4k - allowing it in portrait and landscape mode results in 6k x 6k

Comment (DeepDiver1975) [reply lib/private/Preview/Image.php:78]: I think this is okay this way ..... or is more explanation required here?

Comment (pako81) [reply lib/private/Preview/Image.php:78]: no, that is fine..thx!

**Pull request merged**